go/stats/export_test.go

```diff
@@ -71,38 +71,49 @@ func TestFloat(t *testing.T) {
 	}
 }
 
-func TestInt(t *testing.T) {
+func TestCounter(t *testing.T) {
 	var gotname string
-	var gotv *Int
+	var gotv *Counter
 	clear()
 	Register(func(name string, v expvar.Var) {
 		gotname = name
-		gotv = v.(*Int)
+		gotv = v.(*Counter)
 	})
-	v := NewInt("Int")
+	v := NewCounter("Int", "help")
 	if gotname != "Int" {
 		t.Errorf("want Int, got %s", gotname)
 	}
 	if gotv != v {
 		t.Errorf("want %#v, got %#v", v, gotv)
 	}
-	v.Set(5)
-	if v.Get() != 5 {
-		t.Errorf("want 5, got %v", v.Get())
-	}
 	v.Add(1)
-	if v.Get() != 6 {
-		t.Errorf("want 6, got %v", v.Get())
+	if v.Get() != 1 {
+		t.Errorf("want 1, got %v", v.Get())
 	}
-	if v.String() != "6" {
-		t.Errorf("want 6, got %v", v.Get())
+	if v.String() != "1" {
+		t.Errorf("want 1, got %v", v.Get())
+	}
+	v.Reset()
+	if v.Get() != 0 {
+		t.Errorf("want 0, got %v", v.Get())
 	}
 
-	f := IntFunc(func() int64 {
-		return 1
+}
+
+func TestGaugeFunc(t *testing.T) {
+	var gotname string
```

Comment:
These two variables are unused.

Same for `TestDurationFunc` below.

Also: The `Register() ` here seems to be unnecessary as well?

Can you please double-check this file and report back? Thanks!